autopush-common/src/db/bigtable/pool.rs

```diff
@@ -154,7 +155,11 @@ impl Manager for BigtableClientManager {
     /// `BigtableClient` is the most atomic we can go.
     async fn create(&self) -> Result<BigtableDb, DbError> {
         debug!("🏊 Create a new pool entry.");
-        let entry = BigtableDb::new(self.get_channel()?);
+        let metadata = MetadataBuilder::with_prefix(&self.settings.table_name)
+            .route_to_leader(self.settings.route_to_leader)
```

Comment:
This is currently only used for the health check's `read_rows`, so it also needs a `table_name` routing param